service/src/intTest/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentComponentTest.java

```diff
@@ -95,6 +95,8 @@ public void When_NewAccessDocumentTaskIsStarted_Expect_DatabaseUpdatedAndDocumen
             any(),
             eq(EXPECTED_DOCUMENT_JSON_FILENAME)
         );
+
+        inputStream.close();
```

Comment:
Even if tests it's better to use "try with resources"
```
try (var inputStream = storageConnector.downloadFromStorage(EXPECTED_DOCUMENT_JSON_FILENAME)) {
    ....
}
```

Reply:
Addressed for all instances of 'inputStream.close()'